Modules/ThirdParty/Eigen3/UpdateFromUpstream.sh

```diff
@@ -4,7 +4,7 @@ thirdparty_module_name="Eigen3"
 
 # readonly repo="https://gitlab.kitware.com/third-party/eigen.git"
 upstream_git_url="https://gitlab.kitware.com/phcerdan/eigen.git"
-upstream_git_branch="for/itk-3.3-find_package"
+upstream_git_branch="for/itk-3.4"
```

Comment:
Changes which are not part of Eigen's upstream changes should be in separate commits, such as changes to this file. A recent example:
https://github.com/InsightSoftwareConsortium/ITK/pull/2832